Files/ViewModels/Pages/WidgetsPageViewModel.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace Files.ViewModels.Pages
 {
-    public class YourHomeViewModel : ObservableObject, IDisposable
+    public class WidgetsPageViewModel : ObservableObject, IDisposable
```

Comment:
Can you revert this change? It doesn't appear to be related to the current pull request.